activator/activator.go

```diff
@@ -0,0 +1,90 @@
+package activator
+
+import (
+	"os"
+
+	guuid "github.com/google/uuid"
+
+	"v9_deployment_manager/database"
+	"v9_deployment_manager/log"
+	"v9_deployment_manager/worker"
+)
+
+type Activator struct {
+	driver *database.Driver
+}
+
+func CreateActivator(driver *database.Driver) (*Activator, error) {
+	return &Activator{
+		driver: driver,
+	}, nil
+}
```

Comment:
Why return an `err` if it's always `nil`?

Reply:
good point